CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 ## v42.0.0-SNAPSHOT - unreleased
 
+### 🎁 New Features
+
+* Column-level filtering is now official supported for desktop grids:
+  + `GridModel` now has a `filterModel` config, which accepts a config for the new `GridFilterModel`
+    or a boolean `true` to create the default `GridFilterModel`. `GridFilterModel` is designed to be
+    analogous to `FilterChooserModel`. When a `GridModel` has a `filterModel`, filtering affordances
+    will be added to any column that is marked `filterable`. `GridFilterModel` accepts the following
+    configuration options:
+    + `GridFilterModel.fieldSpecs` - specifies the fields this model supports for filtering. Should
+      be configs for a `GridFilterFieldSpec`. If a `valueSource` is provided, these may be specified
+      as field names in that source or omitted entirely, indicating that all fields should be
+      filter-enabled.
+    + `GridFilterModel.fieldSpecDefaults` - default properties to be assigned to all
+      `GridFilterFieldSpecs` created by this model.
+    + `GridFilterModel.valueSource` - controls which `Store` or cube `View` value suggestions
+      are sourced from. The default `GridFilterModel` sets this to the `GridModel`'s `Store`.
+    + `GridFilterModel.target` - controls which `Store` or cube `View` the generated filter
+      is applied to. The default `GridFilterModel` sets this to the `GridModel`'s `Store`.
+    + `GridFilterModel.initialFilter` - sets an initial filter in the `GridFilterModel`.
+  + New `Column.filterable` to enable a column-level filter affordance in the column header.
+    Note the that the `GridModel` must have a `filterModel`. The filter control offers two tabs - a
+    "Values" tab for enumerative, value-based filter, and a "Custom" tab to build complex filtering
+    queries with multiple clauses.
++ Cube `View` now accepts a `loadModel` property - pass to `Cube.createView()` to link view
+  updates to a `PendingTaskMask`. If `loadModel` is not provided, the `View` will create one.
```

Comment:
*PendingTaskModel*

Reply:
We probably don't need to go in to quite so much detail in the Changelog as to how the API works -- We want to keep it readable and a high level.  I think its fine to point people to the docs of the key classes.

Reply:
Thanks Lee - will streamline it a bit, and make sure all the relevant info can be found in the right class docs